.github/quarkus-github-bot.yml

```diff
@@ -139,16 +139,16 @@ triage:
     - id: hibernate-search
       labels: [area/hibernate-search]
       title: "hibernate.search"
-      notify: [gsmet, yrodiere]
+      notify: [gsmet, marko-bekhta, yrodiere]
```

Comment:
It probably makes sense to add me to the Validator as well (if there's such group)

Reply:
I didn't want to push this on you, but since you're volunteering... :)